swap/src/asb/event_loop.rs

```diff
@@ -319,46 +319,13 @@ where
         min_buy: bitcoin::Amount,
         max_buy: bitcoin::Amount,
     ) -> Result<BidQuote> {
-        let ask_price = self
+        let rate = self
             .latest_rate
             .latest_rate()
-            .context("Failed to get latest rate")?
```

Comment:
There seems to be a rebase error here, we want this functionality :)

Reply:
Apologies yes rebase error!